satcom/openlst/fec.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2023 Robert Zimmerman.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
```

Comment:
Please add a comment like this to help provide traceability. Same for whitening.py

```
#
# This code was copied from https://github.com/rzimmerman/gr-openlst at
# commit 09ba2480de7f5f620f398f387ab402bbf75fa64e
#